pkg/k8shandler/kibana/priorityclass.go

```diff
@@ -0,0 +1,43 @@
+package kibana
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+
+	scheduling "k8s.io/api/scheduling/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+//NewPriorityClass is a constructor to create a PriorityClass
+func NewPriorityClass(priorityclassName string, priorityValue int32, globalDefault bool, description string) *scheduling.PriorityClass {
```

Comment:
kibana doesn't use a priority class... this was leftover from fluentd

Reply:
Fixed by removing `priorityclass.go`